spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -207,5 +207,13 @@ def to_global_id(options = {})
         }.to have_enqueued_job(hello_job).with(42).on_queue("low").at(date).exactly(2).times
       }.to raise_error(message)
     end
+
+    it "throws descriptive error when no test adapter set" do
+      ActiveJob::Base.queue_adapter = :inline
```

Comment:
If this is the last test in this file to get run, we are leaving the queue_adapter as `:inline`. I'm not sure it matters, but I'm always a bit wary of polluting the global environment